src/core/binding.c

```diff
@@ -1618,7 +1625,6 @@ QuicBindingReceive(
         //
         if (!QuicBindingDeliverDatagrams(Binding, SubChain, SubChainLength)) {
             *ReleaseChainTail = SubChain;
-            ReleaseChainTail = SubChainTail;
```

Comment:
I guess I understand that this is unnecessary, but I fear that if any other code added after this that does manipulation of the chain might result in bugs if we remove this. What do you think?

Reply:
Leave it, but comment it out, explaining why? Or `// NOLINT` the line?

Reply:
I'd say `// NOLINT` the line.